src/sentry/static/sentry/app/components/projects/missingProjectMembership.jsx

```diff
@@ -1,33 +1,35 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-
-import createReactClass from 'create-react-class';
+import styled from 'react-emotion';
 
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
 import IndicatorStore from 'app/stores/indicatorStore';
 import {joinTeam} from 'app/actionCreators/teams';
+import HeroIcon from 'app/components/heroIcon';
+import Well from 'app/components/well';
 import withApi from 'app/utils/withApi';
+import space from 'app/styles/space';
 import {t} from 'app/locale';
 
-const MissingProjectMembership = createReactClass({
-  displayName: 'MissingProjectMembership',
-
-  propTypes: {
+class MissingProjectMembership extends React.Component {
+  static propTypes = {
     api: PropTypes.object,
     organization: PropTypes.object.isRequired,
     projectId: PropTypes.string.isRequired,
-  },
+  };
+
+  constructor(props) {
+    super(props);
 
-  getInitialState() {
     const {organization, projectId} = this.props;
     const project = organization.projects.find(p => p.slug === projectId);
 
-    return {
+    this.state = {
       loading: false,
       error: false,
       project,
     };
-  },
+  }
 
   joinTeam(team) {
```

Comment:
This is bound in the event callback

Reply:
In all the places its used there is an existing `.bind()`. I clicked on the 'Join Team' button and it worked.

Reply:
Oops yeah, this was just meant as confirmation